src/data/reducers/shopify.js

```diff
@@ -0,0 +1,271 @@
+const UPDATE_QUANTITY_IN_CART = 'shopify/UPDATE_QUANTITY_IN_CART';
+const REMOVE_LINE_ITEM_IN_CART = 'shopify/REMOVE_LINE_ITEM_IN_CART';
+const OPEN_CART = 'shopify/OPEN_CART';
+const CLOSE_CART = 'shopify/CLOSE_CART';
+const CART_COUNT = 'shopify/CART_COUNT';
+
+const initialState = {
+  isCartOpen: false,
+  cartCount: 0,
+  checkout: {},
+  products: [],
+  featured: [],
+  product: {},
+  shop: {}
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case PRODUCTS_FOUND:
+      return { ...state, products: action.payload };
+    case PRODUCT_FOUND:
+      return { ...state, product: action.payload };
+    case COLLECTION_FOUND:
+      return { ...state, featured: action.payload };
+    case CHECKOUT_FOUND:
+      return { ...state, checkout: action.payload };
+    case SHOP_FOUND:
+      return { ...state, shop: action.payload };
+    case UPDATE_CART_ATTRIBUTE:
+      return { ...state, checkout: action.payload };
+    case ADD_VARIANT_TO_CART:
+      return { ...state, checkout: action.payload };
+    case UPDATE_QUANTITY_IN_CART:
+      return { ...state, checkout: action.payload };
+    case REMOVE_LINE_ITEM_IN_CART:
+      return { ...state, checkout: action.payload };
+    case OPEN_CART:
+      return { ...state, isCartOpen: true };
+    case CLOSE_CART:
+      return { ...state, isCartOpen: false };
+    case CART_COUNT:
+      return { ...state, cartCount: action.payload };
+    default:
+      return state;
+  }
+};
+
+// Gets all the products from Shopify
+const getProducts = () => {
+  return dispatch => {
+    client.product.fetchAll().then(resp => {
+      dispatch({
+        type: PRODUCTS_FOUND,
+        payload: resp
+      });
+    });
+  };
+};
+
+// Gets individual item based on id
+const getProduct = id => {
+  return async dispatch => {
+    const resp = await client.product.fetch(id);
+    dispatch({
+      type: PRODUCT_FOUND,
+      payload: resp
+    });
+    return resp;
+  };
+};
+
+//
+// Gets a  collection based on that collection's id
+//
+// const getCollection = () => {
+// 	const collectionId = "Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzIwODAyMDYwMzAzMw=="
+// 	return (dispatch) => {
+// 		client.collection.fetchWithProducts(collectionId).then((resp) => {
+// 			dispatch({
+// 				type: COLLECTION_FOUND,
+// 				payload: resp.products,
+// 			})
+// 		})
+// 	}
+// }
+
+// Creates initial checkout state from Shopify
+const checkout = () => {
+  return dispatch => {
+    client.checkout.create().then(resp => {
+      dispatch({
+        type: CHECKOUT_FOUND,
+        payload: resp
+      });
+    });
+  };
+};
+
+// Gets Shopify store information
+const shopInfo = () => {
+  return dispatch => {
+    client.shop.fetchInfo().then(resp => {
+      dispatch({
+        type: SHOP_FOUND,
+        payload: resp
+      });
+    });
+  };
+};
+
+// Updates cart attributes
+const updateCartCustomAttributes = (checkoutId, attributes) => {
```

Comment:
Is this where we add the team ID/name and any other attributes we might want to add? under the attributes object?

Reply:
Yes, it will be stored in the checkout object